src/plot_api/plot_api.js

```diff
@@ -2508,6 +2457,391 @@ Plotly.relayout = function relayout(gd, astr, val) {
 };
 
 /**
+ * Animate to a keyframe
+ *
+ * @param {string} name
+ *      name of the keyframe to create
+ * @param {object} animationOpts
```

Comment:
@rreusser now that the API has stabilised, can you fill the jsDoc for `frameOrGroupNameOrFrameList` ?